test/suite/test_compact01.py

```diff
@@ -38,8 +38,7 @@ class test_compact(wttest.WiredTigerTestCase, suite_subprocess):
     name = 'test_compact'
 
     # Use a small page size because we want to create lots of pages.
```

Comment:
Remove this comment, it no longer applies.

Reply:
@ddanderson, should we still say why we are specifying a custom configuration? I am not sure why we need that one.